services/src/atdd/src/test/resources/features/mvp.1/TopologyEventsBasic.feature

```diff
@@ -11,15 +11,15 @@ Feature: Basic Topology Events
   these types of tests - ie ensure we are honoring the policies, and have the ability to overwrite
   the policies.
 
-  @MVP1.1
+  @MVP1
   Scenario: Link is Dropped
 
     Given a clean controller
     And a random linear topology of 5 switches
     When the controller learns the topology
     And multiple links exist between all switches
     And a link is dropped in the middle
-    Then the link disappears from the topology engine.
+    Then the link disappears from the topology engine in 60 seconds.
```

Comment:
why so long time?
can it be decreased by config manipulation?

Reply:
60 seconds is acceptable time length for discovery events defined in TopologyDiscoveryBasic.feature. 
Sure we can decrease it, but must consider the environment where the tests are to be executed.